themes/vocabulary_theme/templates/project_list.html

```diff
@@ -1,66 +1,104 @@
+        <p class="description">This is a list of all active open source software projects that Creative Commons maintains.</p>
+      </div>
+    </div>
+    <div class="body container">
+      <p>Please read the <a href="/contributing-code">Contributing Code</a> page for general information on how to get involved. Contribution guidelines specific to an individual project can be found in the <code>README</code> and the <code>CONTRIBUTING</code> files in the root folder of every repository. If you have any questions, don't hesistate to reach out to us through <a href="/community">one of our community forums</a> or open a GitHub issue.</p>
+      <div class="featured-projects">
+        <h2 class="is-paddingless level-left">
+          Featured Projects
+        </h2>
+        <div class="project-list columns">
+          {% for repo in bag('repos.repos') %}
+          {% if repo.featured %}
+            <div class=" column is-one-third">
+              <article class="card entry-post vertical project-index">
+                <div class="card-content">
+                  <h4 class="card-title b-header"><a href="{{ repo.url }}">{{ repo.english_name }}</a></h4>
+                  <a class="button is-text tiny site-link" href="{{ repo.website }}">
+                    <span>Visit site</span>
+                    <i class="icon external-link"></i>
+                  </a>
+                  <div class="content">
+                    {{ repo.description }}
+                  </div>
+                  <div class="labels">
+                    {% for technology in repo.technologies %}
+                      <button class="button tag">{{ technology }}</button>
+                    {% endfor %}
+                  </div>
+                  <div class="external-links">
+                    <a class="button is-text tiny" href="{{ repo.url }}">
+                      <i class="icon github"></i>
+                      <span class="link-content">Github Repository</span>
+                    </a>
+                    <br>
+                    <a class="button is-text tiny">
+                      <i class="icon slack"></i>
+                      <span class="link-content">Slack channel (#{{ repo.slack }})</span>
+                    </a>
+                  </div>
+                </div>
+              </article>
             </div>
-            </h5>
-            <p class="card-text">{{ repo.description }}</p>
-            {% for technology in repo.technologies %}
-              <span class="badge badge-featured-project mr-1 mb-1">{{ technology }}</span>
-            {% endfor %}
-          </div>
-        </div>
-      {% endif %}
-    {% endfor %}
-  </div>
-  <h2>All Active Projects</h2>
-  <p>This is a complete list of active Creative Commons software projects. Archived projects are not documented here and can be found on the <strong><a href="https://github.com/cc-archive">CC Archive GitHub organization</a>.</strong></p>
-  <table class="table table-striped">
```

Comment:
I think we should work with the [Table container](https://bulma.io/documentation/elements/table/#table-container) from Bulma so we can have the table with horizontal scroll at least.
This is how I see it on phone size:
![Screen Shot 2020-07-14 at 2 54 44 PM](https://user-images.githubusercontent.com/894708/87465157-2eff8580-c5e2-11ea-81a8-a86d080ee9d4.png)

Reply:
on it.